index.js

```diff
@@ -38,12 +41,15 @@ function Scroller(scroller, content, render, isPrepend, isSticky, cb) {
       var m = queue.shift()
       var r = render(m)
       append(scroller, content, r, isPrepend, isSticky)
+      obv.set(queue.length)
     }
   }
 
   function scroll (ev) {
-    if(isEnd(scroller, buffer, isPrepend) || !isFilled(content))
+    if(isEnd(scroller, buffer, isPrepend) || !isFilled(content)) {
       pause.resume()
+      add()
```

Comment:
try taking this out and seeing if things work ok? (or does that kill the top-loading?)